firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -306,39 +325,50 @@ public void onError(Exception error) {
    * chunk-encoded HTTP body. When the connection closes, it attempts to reestablish the stream.
    */
   @SuppressLint("VisibleForTests")
-  synchronized void beginRealtimeHttpStream() {
+  public synchronized void beginRealtimeHttpStream() {
     if (!canMakeHttpStreamConnection()) {
       return;
     }
 
+    int responseCode = 200;
```

Comment:
Did this originally because of the edge case where the client isn't able to make a connection with the server(due to an exception or no connection) and doesn't get a status code back but initialized it to 0 instead and added a check for 0 too.

Reply:
How about checking `responseCode == null` or just `!responseCode` to indicate no response code was set (the connection wasn't made)? 0 is confusing since it's not a valid value

Reply:
Updated it to check for responseCode == null